packages/perseus/src/widgets/label-image/types.ts

```diff
@@ -1,19 +1,21 @@
 // Base marker, with the props that are set by the editor.
-export type MarkerType = {
-    // The list of correct answers expected for the marker.
+export type MarkerAnswers = {
```

Comment:
These types feel off to me. I'm wondering if it's actually something like:

``` TS
// stuff to render
type LabelImageMarker = {
    // Reveal the correctness state of the user selected answers for the marker.
    showCorrectness: "correct" | "incorrect";
    focused: boolean;
    // Translatable Text; The text to show for the marker. Not displayed directly to the user
    label: string;
    // The marker coordinates on the question image as percent of image size.
    x: number;
    y: number;
}

type LabelImageMarkerScoringData = {
    // The list of correct answers expected for the marker. Often only one but can have multiple
    answers: ReadonlyArray<string>;
}

type LabelImageMarkerUserInput = {
    // The user selected list of answers, used to grade the question.
    selected: ReadonlyArray<string>;
}

type LabelImageFullMarker = LabelImageMarker & LabelImageMarkerScoringData & LabelImageMarkerUserInput;
```

Reply:
Updated! Still need to add recommended tests and do additional manual testing though.